src/main/java/picard/analysis/CollectInsertSizeMetrics.java

```diff
@@ -136,7 +136,9 @@ public static void main(final String[] argv) {
 
     @Override protected void setup(final SAMFileHeader header, final File samFile) {
         IOUtil.assertFileIsWritable(OUTPUT);
-        IOUtil.assertFileIsWritable(Histogram_FILE);
+        if (Histogram_FILE != null) {
+            IOUtil.assertFileIsWritable(Histogram_FILE);
+        }
```

Comment:
I think that a LOG.Warning message should be emitted to explain that without this file the PDF will not be generated.